jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -1934,8 +1934,10 @@ ecma_op_object_get_enumerable_property_names (ecma_object_t *obj_p, /**< routine
         return NULL;
       }
 
+      const bool is_enumerable = (prop_desc.flags & ECMA_PROP_IS_ENUMERABLE) != 0;
+      ecma_free_property_descriptor (&prop_desc);
```

Comment:
At this point `status` can be false which means that the requested property does not exist. In this case the property descriptor is uninitialized, therefore must not be freed.

Reply:
At the moment the descriptors are empty initialized by default.

Reply:
Indeed, just ignore my comment.